modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -1,11 +1,17 @@
-import Base from 'elementor/assets/dev/js/frontend/handlers/base';
+import NestedTabsNew from './nested-tabs-new';
 import {
 	changeScrollStatus,
 	setHorizontalScrollAlignment,
 	setHorizontalTitleScrollValues,
-} from 'elementor/assets/dev/js/frontend/utils/flex-horizontal-scroll';
+} from 'elementor-frontend-utils/flex-horizontal-scroll';
+
+export default class NestedTabs extends NestedTabsNew {
+	__construct( settings ) {
+		super.__construct( settings );
+
+		this.isHtmlExperimentActive = elementorFrontendConfig.experimentalFeatures[ 'nested-elements-html' ];
```

Comment:
The idea is to check per method inside this class if the old or the new version will be used.

All the conditional logic for the experiment will be added in the old file, so that once the experiment will be removed, we only have to replace this handler file with `nested-tabs-new.js`, without having to make any changes.